core/commands/ls.go

```diff
@@ -49,7 +49,7 @@ var LsCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline: "List directory contents for Unix filesystem objects.",
 		ShortDescription: `
-Displays the contents of an IPFS or IPNS object(s) at the given path, with
+Displays the contents of an IPFS or IPNS object(s) at the given path, (Won't work with MFS paths - Use 'ipfs files ls' instead) with
```

Comment:
I'm still not convinced this helps anyone and I'd rather not introduce a new concept (mfs) here. Did you actually experience confusion expecting `ipfs ls` to list MFS paths?